.changelog/3052.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+runner: start health check before blocking on adoption
```

Comment:
Don't need to changelog this, cause its only a bug with the runner adoption which was never released.